### PR TITLE
Update changelog.md for Nuget package release

### PR DESCRIPTION
**Pull request by mwesigwaguma**
Related Pull request: https://github.com/Azure/azure-sdk-for-net/pull/36990

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by a-santamaria — APPROVED** [review]

File: sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/CHANGELOG.md [hunk 1]
```diff
@@ -10,12 +10,26 @@
 
 ### Other Changes
 
-## 1.1.0-beta.2 (2023-06-21)
+## 1.1.0-beta.2 (2023-06-21)### Features Added
```
Comment (ArthurMa1978) [inline sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/CHANGELOG.md:13]: Please remove the "### Features Added" in this line

Comment (mwesigwaguma) [reply sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/CHANGELOG.md]: updated

**Comment by mwesigwaguma** [comment]
> **API change check**
> 
> APIView has identified API level changes in this PR and created following API reviews.
> 
> [Azure.ResourceManager.ServiceFabricManagedClusters](https://apiview.dev/Assemblies/Review/582a13da375d4a5dbbac94141a03db12)

Hey @ArthurMa1978 can you take a look at this

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by ArthurMa1978** [comment]
> > **API change check**
> > APIView has identified API level changes in this PR and created following API reviews.
> > [Azure.ResourceManager.ServiceFabricManagedClusters](https://apiview.dev/Assemblies/Review/582a13da375d4a5dbbac94141a03db12)
> 
> Hey @ArthurMa1978 Arthur Ma FTE is the word 'beta' needed in the version name ? it was automatically generated but I would like to change it to just 1.1.0.0. Also this is the version they here: https://apiview.dev/Assemblies/Review/582a13da375d4a5dbbac94141a03db12# Also, it looks like merging is still blocked.

Here is the versioning policy https://azure.github.io/azure-sdk/policies_releases.html#net

**Pull request merged**